internal-packages/hub-api-client/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "@quri/hub-api-client",
```

Comment:
This package is used by `apps/hub` and by `internal-packages/ai`. Previously we had some amount of copy-paste there.

The `HubApiClient` class can be parameterized by the domain, which will allow us to implement features like "copy this squiggle model from production to my local dev instance" in the future. (It's already used by `/admin/seed` page for this purpose, for fetching `ozziegooen/sTest` library).

This package can potentially be published on NPM - it's tiny and doesn't depend on anything else.